compiler/src/iree/compiler/API/Internal/CompilerDriver.cpp

```diff
@@ -90,6 +89,74 @@
 namespace mlir::iree_compiler::embed {
 namespace {
 
+// If note using memfd_create, then we need to align output buffers
```

Comment:
nit: did you mean 'not'?
```suggestion
// If not using memfd_create, then we need to align output buffers
```